docker/nginx.conf

```diff
@@ -22,9 +22,7 @@ http {
         location / {
             root /opt/nginx/root;
             index index.html;
-            # This line has been adjusted for the temporary different external URL.
-            # Don't include this change in the main branch.
-            rewrite ^/https-everywhere($|//+)(.*) /https-everywhere-2021/$2 permanent;
+            rewrite ^/https-everywhere-2021($|//+)(.*) /https-everywhere-2021/$2 permanent;
```

Comment:
Unfortunately this is where it gets kind of gnarly: our ingress transforms all external requests (`/https-everywhere-2021/`, or, formerly, `/https-everywhere/`) to internal requests for `/https-everywhere/`, because that's what's being served by this container (files are copied to `/opt/nginx/root/https-everywhere/`). So we need to leave this rewrite as is for it to do its thing.

Reply:
Roger that, will amend and at least clarify the comment as a warning to posterity.